msm/src/lib.rs

```diff
@@ -180,7 +211,7 @@ mod tests {
         {
             let mut proof_clone = proof.clone();
             proof_clone.proof_evals.witness_evals = proof_prime.proof_evals.witness_evals;
-            let verifies = verify::<_, OpeningProof, BaseSponge, ScalarSponge, N>(
+            let verifies = verify::<_, OpeningProof, BaseSponge, ScalarSponge, MSM_FFADD_N_COLUMNS>(
```

Comment:
We should add the reason the verification does not pass: for a follow-up.
It can be for multiple reasons that the verification does not pass, not only because of the constraints.